### DISCUSSION TITLE
Error loading Rowy after I delete Firebase project in Firebase

### DISCUSSION THREAD

**Discussion by gsleininger** [AntlerVC/firetable#930] [Support / Q&A] [answered]
1) Create a project in Rowy connected to a Firebase project.
2) Delete the Firebase project from Google Console.
3) Try to load the project in Rowy or access its settings (in order to delete it in Rowy).
4) Error page. "Something went wrong"

**Comment by BeeBombshell** [comment, marked as answer]
Hi @gsleininger. We have added a delete project button at a workspace-level setting recently. You can delete your project from the workspace settings. I hope this resolves the issue.

<img width="561" alt="image" src="https://user-images.githubusercontent.com/53828745/201162578-4f6ef929-7ca7-4c7b-8291-3339a4e5040b.png">
